doc/faq.rst

```diff
@@ -280,6 +280,19 @@ program: Insert the following instructions in your main script::
 You can find more default on the new start methods in the `multiprocessing
 documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods>`_.
 
+Why does my job use more cores than specified with n_jobs under OSX or Linux?
+-----------------------------------------------------------------------------
+This can happen when the parallelized computation is handled outside of Python by another library,
+such as MKL or OpenBLAS. In that case, the value of ``n_jobs`` is ignored by the library,
```

Comment:
Scikit-learn still adheres to n_jobs, but vectorized numpy operations will make use of multiple threads within each scikit-learn job (thread or process)

Reply:
Thanks for the clarification. I’ll reword the instructions.